runXcodeTests.sh

```diff
@@ -11,15 +11,8 @@ if [ -z "$1" ]
 fi
 
 xctool \
```

Comment:
There’s no such xcode project, so this script failed to run. Or is that auto-generated somewhere that I overlooked? (Couldn’t find anything about it in the docs.)